tests/unit/test_interface.py

```diff
@@ -135,8 +137,12 @@ def test_set_up_dask_client_default() -> None:
                 "assigned via an srun (both for pytests as well as actual work)."
             )
     else:
-        available_memory = psutil.virtual_memory().total / (1024 ** 3)
-    assert np.isclose(sum(worker["memory_limit"] / 1024**3 for worker in workers.values()), available_memory, rtol=0.01)
+        available_memory = psutil.virtual_memory().total / (1024**3)
+    assert np.isclose(
+        sum(worker["memory_limit"] / 1024**3 for worker in workers.values()),
```

Comment:
Is this checking we were close to an expected memory that was profiled or something?

Reply:
It's checking that the memory of the dask cluster is close to what we expected when setting up the dask cluster